src/docs/content/disaster-recovery/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Disaster Recovery"
+date: 2021-09-02 T16:04:15-06:00
+draft: false
+weight: 510
+showChildren: true
+---
+
+## Introduction
+
+In this section we will cover the disaster recovery solution for MuShop deployed on OCI (Oracle Cloud Infrastructure).
```

Comment:
This is a tad of a nit-pick. 
Try to avoid use of the first person (`we`) in the voice & tone in these docs. A less formal and reader-friendly tone is encouraged, but the imperatives and benefits would take precedence.